cmd/dp-recipe-api/main.go

```diff
@@ -64,4 +67,36 @@ func main() {
 		api.CreateAndInitialiseRecipeAPI(ctx, *cfg, store.DataStore{Backend: nil})
 	}
 
+	apiErrors := make(chan error, 1)
+
+	// Gracefully shutdown the application closing any open resources.
+	gracefulShutdown := func() {
+		log.Event(ctx, (fmt.Sprintf("shutdown with timeout: %s", cfg.GracefulShutdownTimeout)), log.INFO)
```

Comment:
no need for the braces around `fmt.Sprintf`

Reply:
Done in commit "Simplifying code and improving gracefulShutdown."

Reply:
Done in PR https://github.com/ONSdigital/dp-recipe-api/pull/127